projects/experimental/components/pdf-viewer/pdf-viewer.style.less

```diff
@@ -0,0 +1,80 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+tui-pdf-viewer {
+    display: block;
+    inline-size: 100%;
+    block-size: 100vh;
```

Comment:
This would cause issues when address bar is present. Also in SheetDialog mode. A little unnecessary scroll is there even in page dialog on mobile.